packages/common/src/jsonApiClient/modify/inverseUpdateRelationship.js

```diff
@@ -0,0 +1,180 @@
+const { Dependor } = require('../../Dependor')
+const buildOperationId = require('../../buildOperationId')
+
+const dep = new Dependor({
+  buildOperationId,
+})
+
+const deleteNotIncludedRelationships = ({
+  inverseUpdateOperationId,
+  item,
+  log,
+  openApiClient,
+  relationItemsToUpdate,
+  relationKey,
+  type,
+}) => {
+  const getRelationshipOperationId = buildOperationId({
+    operationType: 'getRelationship',
+    relationKey,
+    resource: type,
+  })
+
+  const request = {
+    pathParams: {
+      id: item.id,
+    },
+  }
+  log.debug(
+    `Fetching current relationships with getRelationshipOperationId: ${
+      getRelationshipOperationId
+    }`,
+    request
+  )
+  return openApiClient
+    .call(getRelationshipOperationId, request)
+    .then(result => {
+      const { data: existingRelations } = result
+      if (!existingRelations) {
+        log.debug(`No existing relations for ${relationKey}`)
+        return Promise.resolve({ data: null })
+      }
+
+      log.debug('Got existing relations:', existingRelations)
+
+      const existingRelationsArray = Array.isArray(existingRelations)
+        ? existingRelations
+        : [existingRelations]
+
+      const relationItemsToUpdateArray = Array.isArray(relationItemsToUpdate)
+        ? relationItemsToUpdate
+        : [relationItemsToUpdate]
+
+      const relationsToRemove = existingRelationsArray.filter(
+        ({ id: existingId }) => {
+          const existInUpdate = relationItemsToUpdateArray.find(({ id }) => {
+            return id === existingId
+          })
+          if (existInUpdate) {
+            log.debug(
+              `Not removing current relation id: ${existingId}. Exist in update`
+            )
+          }
+
+          return !existInUpdate
+        }
+      )
+
+      if (relationsToRemove.length) {
+        log.debug(
+          `The following ${relationKey} should be removed:`,
+          relationsToRemove
+        )
+
+        const updateRelationshipOperationId = buildOperationId({
```

Comment:
this is not used (only in the log in line 92, but it should be `inverseUpdateOperationId`